injected/src/services/plugins.ts

```diff
@@ -7,6 +7,16 @@ interface Entrypoint {
   menu: boolean;
 }
 
+export interface StorePlatforms {
+  linux: Platform;
+  windows: Platform;
+  mac: Platform;
```

Comment:
mac is returned from the function as `darwin`. Probably should make a translation method to translate the platform into something we want, or encode the translation as an enum